packages/annotated/dev-scripts/README.md

```diff
@@ -0,0 +1 @@
+# @annotated/dev-scripts
```

Comment:
Also TODO? Overall I think that the structure of this lends well to us having strong component-based documentation, but I don't know how much of that you'd prefer to offload into the storybook (in which case perhaps we can do some kind of generator for these readmes, or via docusaurus?)

Reply:
yeah I think we need to figure out where to go from here. I do like how tools like radix and blueprintjs do their documentation, e.g. https://www.radix-ui.com/docs/primitives/components/switch

[Docusaurus 2.0 has MDX integration](https://docusaurus.io/docs/next/markdown-features/react) with could allow us to write markdown docs with inline components. I'm leaning towards this option.

Definitely need to have a doc day soon lol